owncloudDomain/src/test/java/com/owncloud/android/domain/user/model/UserQuotaTest.kt

```diff
@@ -30,25 +30,32 @@ class UserQuotaTest {
         val item = UserQuota(
             "",
             800,
-            200
+            200,
+            1000,
+            "normal"
```

Comment:
Is this a possible state?

Reply:
Yeh, `normal` state is when the quota is limited but the value is under 75% of total